docs/fiddles/features/progress-bar/index.html

```diff
@@ -7,10 +7,12 @@
 </head>
 <body>
     <h1>Hello World!</h1>
-    <p>
-        We are using node <script>document.write(process.versions.node)</script>,
-        Chrome <script>document.write(process.versions.chrome)</script>,
-        and Electron <script>document.write(process.versions.electron)</script>.
-    </p>
+    <p>Keep an eye on the Dock (Mac)/Task Bar (Windows) for this application!</p>
+    <p>It should indicate a progress that advances from 0 to 100%.</p>
+    <p>For the next 5 seconds, on Windows, it should remain in an indeterminate state, while on
+      Mac it should pin at 100%.</p>
+    <p>Finally, it should disappear entirely.</p>
```

Comment:
small suggestions: not sure if different paragraphs is right here? Maybe just `<ol><li>...</li></ol>` or something?

```suggestion
    <ol>
        <li>It should indicate a progress that advances from 0 to 100%.</li>
        <li>For the next 5 seconds, on Windows, it should remain in an indeterminate state, while on Mac it should pin at 100%.</li>
        <li>Finally, it should disappear entirely.</li>
    </ol>
```

Reply:
Thanks. I think I'll stick with the paragraphs.